### ISSUE TITLE
I installed Deconz 5.2 and is not showing up as on my local host

### ISSUE THREAD

**Issue by eskimos2000** [home-assistant/hassio-addons#1105]
Hi there,

I am stragling with that issue for some time I tried to make fresh installation of deconz addon. Removed old on restarted server and installed new one my configs:

`Config
1
device: /dev/ttyUSB0
2
vnc_password: ''
3
dbg_aps: 1
4
​`
`

Container | Host | Description
-- | -- | --
40850/tcp |   | deCONZ API backend (Not required for Ingress)
5900/tcp |   | deCONZ graphical desktop via VNC
8081/tcp |   | deCONZ WebSocket (Not required for Ingress)

`

On the log file I am getting constant flow of this info:

`14:55:14:735 APS-DATA.request id: 53, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:55:14:922 APS-DATA.confirm id: 53, status: 0x00 SUCCESS
14:55:14:922 APS-DATA.confirm request id: 53 -> confirmed, timeout 10899032
14:55:14:953 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:55:14:954 APS-DATA.indication request id: 53 -> finished
14:55:14:954 APS-DATA.request id: 53 erase from queue
14:55:22:566 Current channel 15
14:55:22:598 Device TTL 1931 s flags: 0x7
14:55:31:055 APS-DATA.request id: 55, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:55:31:126 APS-DATA.confirm id: 55, status: 0x00 SUCCESS
14:55:31:126 APS-DATA.confirm request id: 55 -> confirmed, timeout 10898664
14:55:31:158 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:55:31:158 APS-DATA.indication request id: 55 -> finished
14:55:31:158 APS-DATA.request id: 55 erase from queue
14:55:38:947 APS-DATA.indication srcAddr: 0xd0ba, srcEp: 0x01 dstAddrMode: 2, profile: 0x0104, cluster: 0x0402, lqi: 255, rssi: -22
14:55:38:948 ZCL attribute report 0x00158D00030A049C for cluster 0x0402, ep 0x01
14:55:38:979 APS-DATA.indication srcAddr: 0xd0ba, srcEp: 0x01 dstAddrMode: 2, profile: 0x0104, cluster: 0x0405, lqi: 255, rssi: -22
14:55:38:981 ZCL attribute report 0x00158D00030A049C for cluster 0x0405, ep 0x01
14:55:39:011 APS-DATA.indication srcAddr: 0xd0ba, srcEp: 0x01 dstAddrMode: 2, profile: 0x0104, cluster: 0x0403, lqi: 255, rssi: -22
14:55:39:013 ZCL attribute report 0x00158D00030A049C for cluster 0x0403, ep 0x01
14:55:47:375 APS-DATA.request id: 63, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:55:47:441 APS-DATA.confirm id: 63, status: 0x00 SUCCESS
14:55:47:441 APS-DATA.confirm request id: 63 -> confirmed, timeout 9641680
14:55:47:473 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 255, rssi: -22
14:55:47:473 APS-DATA.indication request id: 63 -> finished
14:55:47:473 APS-DATA.request id: 63 erase from queue
14:56:03:695 APS-DATA.request id: 65, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:56:03:757 APS-DATA.confirm id: 65, status: 0x00 SUCCESS
14:56:03:757 APS-DATA.confirm request id: 65 -> confirmed, timeout 11396024
14:56:03:789 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:56:03:789 APS-DATA.indication request id: 65 -> finished
14:56:03:789 APS-DATA.request id: 65 erase from queue
14:56:20:015 APS-DATA.request id: 67, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:56:20:089 APS-DATA.confirm id: 67, status: 0x00 SUCCESS
14:56:20:089 APS-DATA.confirm request id: 67 -> confirmed, timeout 11397104
14:56:20:121 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:56:20:121 APS-DATA.indication request id: 67 -> finished
14:56:20:121 APS-DATA.request id: 67 erase from queue
14:56:22:573 Current channel 15
14:56:22:605 Device TTL 1871 s flags: 0x7
14:56:36:335 APS-DATA.request id: 69, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:56:36:405 APS-DATA.confirm id: 69, status: 0x00 SUCCESS
14:56:36:405 APS-DATA.confirm request id: 69 -> confirmed, timeout 8829264
14:56:36:437 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:56:36:437 APS-DATA.indication request id: 69 -> finished
14:56:36:437 APS-DATA.request id: 69 erase from queue
14:56:42:849 GW firmware version: 0x26330500
14:56:42:849 GW firmware version is up to date: 0x26330500
14:56:52:447 APS-DATA.request id: 71, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:56:52:512 APS-DATA.confirm id: 71, status: 0x00 SUCCESS
14:56:52:512 APS-DATA.confirm request id: 71 -> confirmed, timeout 8829584
14:56:52:544 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 130, rssi: 0
14:56:52:544 APS-DATA.indication request id: 71 -> finished
14:56:52:544 APS-DATA.request id: 71 erase from queue
14:57:08:479 APS-DATA.request id: 73, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:57:08:539 APS-DATA.confirm id: 73, status: 0x00 SUCCESS
14:57:08:539 APS-DATA.confirm request id: 73 -> confirmed, timeout 6246320
14:57:08:571 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:57:08:571 APS-DATA.indication request id: 73 -> finished
14:57:08:571 APS-DATA.request id: 73 erase from queue
14:57:22:579 Current channel 15
14:57:22:611 Device TTL 1811 s flags: 0x7
14:57:24:798 APS-DATA.request id: 75, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:57:24:871 APS-DATA.confirm id: 75, status: 0x00 SUCCESS
14:57:24:871 APS-DATA.confirm request id: 75 -> confirmed, timeout 11422824
14:57:24:903 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:57:24:903 APS-DATA.indication request id: 75 -> finished
14:57:24:903 APS-DATA.request id: 75 erase from queue
14:57:41:119 APS-DATA.request id: 77, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:57:41:187 APS-DATA.confirm id: 77, status: 0x00 SUCCESS
14:57:41:187 APS-DATA.confirm request id: 77 -> confirmed, timeout 11393976
14:57:41:219 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:57:41:219 APS-DATA.indication request id: 77 -> finished
14:57:41:219 APS-DATA.request id: 77 erase from queue
14:57:54:057 APS-DATA.indication srcAddr: 0x07bc, srcEp: 0x01 dstAddrMode: 2, profile: 0x0104, cluster: 0x0000, lqi: 255, rssi: -22
14:57:54:058 ZCL attribute report 0x00158D00024919DE for cluster 0x0000, ep 0x01
14:57:54:058 0x00158D00024919DE extract Xiaomi special attribute 0xFF01
14:57:54:058 	64 on/off 1
14:57:54:058 	03 temperature 41 °C
14:57:54:058 	98 power (?) 0x410A5A1D
14:57:54:058 	95 consumption (?) 0x41A57939
14:57:54:058 	96 unsupported tag (data type 0x23)
14:57:54:058 	05 RSSI dB (?) 23 (0x0017)
14:57:54:058 	9a unknown 0 (0x00)
14:57:54:058 	08 unknown 4886 (0x1316)
14:57:54:058 	07 unknown 0 (0x0000000000000000)
14:57:54:058 	09 unknown 513 (0x0201)
14:57:57:439 APS-DATA.request id: 81, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:57:57:502 APS-DATA.confirm id: 81, status: 0x00 SUCCESS
14:57:57:502 APS-DATA.confirm request id: 81 -> confirmed, timeout 6244504
14:57:57:534 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 255, rssi: -22
14:57:57:534 APS-DATA.indication request id: 81 -> finished
14:57:57:534 APS-DATA.request id: 81 erase from queue
14:58:13:758 APS-DATA.request id: 83, addrmode: 0x03, addr: 0x00212effff017192, profile: 0x0000, cluster: 0x0031, ep: 0x00 -> 0x00 queue: 0 len: 2 tx.options 0x00
14:58:13:834 APS-DATA.confirm id: 83, status: 0x00 SUCCESS
14:58:13:834 APS-DATA.confirm request id: 83 -> confirmed, timeout 11394128
14:58:13:866 APS-DATA.indication srcAddr: 0x0000, srcEp: 0x00 dstAddrMode: 2, profile: 0x0000, cluster: 0x8031, lqi: 0, rssi: 74
14:58:13:866 APS-DATA.indication request id: 83 -> finished
14:58:13:866 APS-DATA.request id: 83 erase from queue
14:58:22:569 Current channel 15`

I have notice that on phoscan app gateway is as 172.30.33.2 but usually it was my local host 192.168.1.250. Moreover internal deylight sensor is not reachable. 
However on  Phoscan login page it is showing as on my local ip address.

I had this problem when I tried to upgrade from 3.8 to 5.1 but I thought that with fresh installation should be all good but it is not. Any help please?

**Comment by pvizeli** [comment]
You need remove the integration from Home Assistant and use add it again with add-on discovery

**Comment by eskimos2000** [comment]
I did it, forgot to mentioned. I did remove old add-on, removed integration, deleted db, restarted HA, installed new doconz, and it is still showing as 172...... I do not know why?. I tried to add integration, went ok, but integration is empty due to that wrong IP address. Very weird. When I restore from backup my second 3.8 last which is working for me everything is ok, phoscan, integration. So I do not know.
